viewer/src/datamodels/cad/sector/CachedRepository.ts

```diff
@@ -437,11 +411,36 @@ export class CachedRepository implements Repository {
     return meshesGroupedByFile;
   }
 
-  private cacheKey(wantedSector: WantedSector) {
+  private wantedSectorCacheKey(wantedSector: WantedSector) {
     return '' + wantedSector.blobUrl + '.' + wantedSector.metadata.id + '.' + wantedSector.levelOfDetail;
   }
 
-  private ctmKey(request: { blobUrl: string; fileName: string }) {
+  private updateConsumedSectorCache(wantedSector: WantedSector, networkObservable?: Observable<ConsumedSector>) {
```

Comment:
Added two functions instead of internalizing cleanup in memorycache, for both ctm and consumedsector caches.